vendor/opam/src/core/dune

```diff
@@ -0,0 +1,45 @@
+(library
+  (name        opam_core)
+  (synopsis    "OCaml Package Manager core internal stdlib")
+  (libraries   dune_re unix bigarray sha uutf
+               (select opamACL.ml from
+                       (opam-core.libacl -> opamACL.libacl.ml)
+                       (                 -> opamACL.dummy.ml))
+               (select opamStubs.ml from
+                       (opam-core.stubs -> opamStubs.win32.ml)
+                       (                -> opamStubs.dummy.ml)))
+  (flags       (:standard
+               (:include ../ocaml-flags-standard.sexp)))
+  (wrapped     false))
+
+(rule
+  (with-stdin-from opamCompat.pp.ml
+                   (with-stdout-to opamCompat.ml
+                                   (run ./gen/gen.exe))))
+(rule
+  (with-stdin-from opamCompat.pp.mli
+                   (with-stdout-to opamCompat.mli
+                                   (run ./gen/gen.exe))))
+
+(rule
+  (copy opamStubsTypes.ml opamStubsTypes.mli))
+
+
+(rule
+  (write-file opamCoreConfigDeveloper.ml
+    "let value = \"%{read-strings:developer}\""))
+
+(rule
+  (write-file opamVersionInfo.ml
```

Comment:
we can replace all this version stuff with some hard coded version file. It should never appear anywhere user facing anyway.

Reply:
All of the other `rule` stanzas should be removed as well. None of them are usable in bootstrap.